src/XMakeCommandLine/UnitTests/project.json

```diff
@@ -7,28 +7,25 @@
   },
   "frameworks": {
     "net451": {
-      "dependencies": {
-        "xunit.runner.visualstudio": "2.1.0"
-      }
+      "dependencies": {}
     },
     "net46": {
-      "dependencies": {
-        "xunit.runner.visualstudio": "2.1.0"
```

Comment:
We do tests for full framework with the console runner, too.  But having them in VS is pretty nice . . .

Agree that this isn't blocking.